global.json

```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "3.0.100-preview6-012264",
+    "dotnet": "3.0.100-preview7-012512",
```

Comment:
trail blazing!

Reply:
do we need to update to preview7 due to the CAS changes?

Reply:
no, to get the new NetSdkWarning strings.